### PR TITLE
feat(import account): recover account from phrases you input

### PR DESCRIPTION
**Pull request by genie-magic**
**What this PR does** 📖
Recover account from phrases you input

**Which issue(s) this PR fixes** 🔨

Fixes # AP-15

**Special notes for reviewers** 🗒️
Test Case for reviewers:
1. create an account
2. while creating an account, you should save recover phrases
image
3. after you register and login.. clear localStorage and go to '/'
4. input pin and click import account
5. input recover phrases you saved
6. after clicking import account you can log in with your created account

### CODE REVIEW

File: store/accounts/actions.ts [hunk 1]
```diff
@@ -99,6 +99,28 @@ export default {
 
     commit('setEncryptedPhrase', encryptedPhrase)
   },
+  /**
+   * @method generateWallet DocsTODO
```
Comment (iltumio) [inline store/accounts/actions.ts:103]: wrong method name

File: pages/setup/InputAccount/index.vue [hunk 2]
```diff
@@ -23,8 +23,21 @@ export default Vue.extend({
     },
   },
   methods: {
-    recoverAccount() {
-      /* recover account action will be implemented on bip39 service ticket */
+    /**
+     * @method recoverAccount DocsTODO
+     * @description recover account with 12 mnemonic recover phrases
+     * @param
```
Comment (iltumio) [inline pages/setup/InputAccount/index.vue:29]: remove @param since is not needed

File: pages/setup/InputAccount/index.vue [hunk 3]
```diff
@@ -23,8 +23,21 @@ export default Vue.extend({
...
+     * @example
```
Comment (iltumio) [inline pages/setup/InputAccount/index.vue:30]: provide an example like
```
this.recoverAccount()
```

**Review by iltumio — CHANGES_REQUESTED** [review]
The functionality is ok. There are some errors in the comments to be fixed.
From the UX point of view the component does not allow to paste a whole phrase at once. We should open a separate ticket for that

**Comment by iltumio** [comment]
@RetroPronghorn we should allow users to paste the whole phrase at once otherwise they have to input every single word by hand that is not user friendly

**Review by stavares843 — CHANGES_REQUESTED** [review]
Captura de ecrã 2021-12-08, às 15 36 29

on the seed, appeared the same word 2x, birth birth

but when trying to add birth 2x, is not allowed because is already added

check hogan's comment [here](https://discord.com/channels/773445116572073995/859670812856418314/918165565984632872) regarding how to address it

**Comment by genie-magic** [comment]
> Captura de ecrã 2021-12-08, às 15 36 29
> 
> on the seed, appeared the same word 2x, birth birth
> 
> but when trying to add birth 2x, is not allowed because is already added
> 
> check hogan's comment [here](https://discord.com/channels/773445116572073995/859670812856418314/918165565984632872) regarding how to address it

I understand you.. yes. that is problem but not issue on this PR.. that is issue when creating account which populate same word.
What do you think abou that @WanderingHogan ?  We can make separate ticket or something but for this PR, we can skip that.

**Comment by stavares843** [comment]
hogan already had replied that [here](https://discord.com/channels/773445116572073995/859670812856418314/918165565984632872)


to address that on this PR

**Comment by WanderingHogan** [comment]
Alright - so changes -
please make the changes Manuel requested
please make it so when a word is selected it is no longer removed from the list of words, and so that there is no limit on the number of times it is selected in the words list up to the max number of words that can be selected

**Comment by genie-magic** [comment]
@WanderingHogan , @iltumio , @stavares843 
I fixed all you mentioned. plz check and let me know.

**Comment by stavares843** [comment]
great, thanks! 🎉 checking

**Comment by stavares843** [comment]
Captura de ecrã 2021-12-08, às 22 54 42

getting the above error trying to import

**Comment by WanderingHogan** [comment]
@stavares843 I was wrong - the words do have to be in order. We should update the text so the user knows to enter them in order. This works greats when they are in the correct order.

**Comment by WanderingHogan** [comment]
Language changed to tell user to put words in order & error message added in the translations file.

**Comment by stavares843** [comment]
thanks for clarifying!

**Comment by stavares843** [comment]
thanks for adding the locales! 💥

**Review by stavares843 — APPROVED** [review]
approved ✅

**Review by iltumio — APPROVED** [review]

**Pull request merged**